device/edge-sample/edge_sample_module.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var Protocol = require('azure-iot-device-mqtt').Mqtt;
+var Client = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+var fs = require('fs');
+
+var connectionString = process.env.EdgeHubConnectionString;
+
+var client = Client.fromConnectionString(connectionString, Protocol);
+console.log('got client');
+
+client.on('error', function (err) {
+  console.error(err.message);
+});
+
+client.setOptions({
+  ca: fs.readFileSync(process.env.EdgeModuleCACertificateFile).toString('ascii')
```

Comment:
>EdgeModuleCACertificateFile [](start = 34, length = 27)

nit: that's unusual casing/format for an env variable. usually see them all uppercase with underscores EDGE_CA_CERT for example

Reply:
That's the convention that they use in Edge containers.  I don't like it either, but I didn't invent it.

---
In reply to: [178390273](https://github.com/Azure/azure-iot-sdk-node/pull/260#discussion_r178390273) [](ancestors = 178390273)